NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/InvitePlayer.java

```diff
@@ -74,9 +74,12 @@ public boolean execute(CommandSender s, String[] args) {
 			}
 			return true;
 		}
+		if (pType == PlayerType.NOT_BLACKLISTED) {
+			p.sendMessage(ChatColor.RED + "I think we both know that this shouldnt be possible");
```

Comment:
No, this is someone trying to explicitly invite someone to the NOT_BLACKLISTED playertype, which should never work because by default everyone who is not blacklisted is included in that playertype and being able to add players to this playertype would lead to a lot of weird issues. I made the error message that way, because the system kinda implies that this shouldn't be possible, but I can change it to make it a bit more user friendly.